README.rst

```diff
@@ -178,62 +178,86 @@ GetComponentImages example:
+            # box is a dict with x, y, w and h keys (w.r.t. full image)
+            img.save('phototest_line{0}_{x}x{y}+{w}x{h}.png'.format(i, **box), format='PNG')
 
-Orientation and script detection (OSD):
-```````````````````````````````````````
+GetIterator example:
+````````````````````
 
 .. code:: python
 
     from PIL import Image
-    from tesserocr import PyTessBaseAPI, PSM
+    from tesserocr import PyTessBaseAPI, RIL
 
-    with PyTessBaseAPI(psm=PSM.AUTO_OSD) as api:
-        image = Image.open("/usr/src/tesseract/testing/eurotext.tif")
+    image = Image.open('/usr/src/tesseract/testing/phototest.tif')
+    with PyTessBaseAPI() as api:
         api.SetImage(image)
         api.Recognize()
+        
+        it = api.GetIterator()
+        for line in iterate_level(it, RIL.TEXTLINE):
+            text = line.GetUTF8Text(RIL.TEXTLINE)
+            conf = line.Confidence(RIL.TEXTLINE)
+            bbox = line.BoundingBox(RIL.TEXTLINE)
+            bbox = {'x': int(bbox[0]),
+                    'y': int(bbox[1]),
+                    'w': int(bbox[2])-int(bbox[0]),
+                    'h': int(bbox[3])-int(bbox[1])}
+            print(u"Box[{0}]: x={x}, y={y}, w={w}, h={h}, "
+                  "confidence: {1}, text: {2}".format(i, conf, text, **bbox))
 
-        it = api.AnalyseLayout()
-        orientation, direction, order, deskew_angle = it.Orientation()
-        print("Orientation: {:d}".format(orientation))
-        print("WritingDirection: {:d}".format(direction))
-        print("TextlineOrder: {:d}".format(order))
-        print("Deskew angle: {:.4f}".format(deskew_angle))
-
-or more simply with ``OSD_ONLY`` page segmentation mode:
+Layout analysis with orientation and deskewing:
+```````````````````````````````````````````````
 
 .. code:: python
 
+    import math
+    from PIL import Image
     from tesserocr import PyTessBaseAPI, PSM
+    from tesserocr import Orientation, WritingDirection, TextlineOrder
 
-    with PyTessBaseAPI(psm=PSM.OSD_ONLY) as api:
-        api.SetImageFile("/usr/src/tesseract/testing/eurotext.tif")
-
-        os = api.DetectOS()
-        print("Orientation: {orientation}\nOrientation confidence: {oconfidence}\n"
-              "Script: {script}\nScript confidence: {sconfidence}".format(**os))
+    with PyTessBaseAPI(psm=PSM.AUTO) as api:
+        image = Image.open("/usr/src/tesseract/testing/eurotext.tif")
+        api.SetImage(image)
 
-more human-readable info with tesseract 4+ (demonstrates LSTM engine usage):
+        it = api.AnalyseLayout()
+        orientation, direction, order, deskew_angle = it.Orientation()
+        print("Orientation: {}".format(membername(Orientation, orientation)))
+        print("WritingDirection: {}".format(membername(WritingDirection, direction)))
+        print("TextlineOrder: {:d}".format(membername(TextlineOrder, order)))
+        print("Deskew angle: {:.1f}°".format(deskew_angle * 180 / math.pi))
```

Comment:
Seems more informative to me to get `left-to-right` or `PAGE_UP` instead of just 0. Also converting the angle from radians to degrees is more illustrative.